web/controllers/rules/helpers.ex

```diff
@@ -18,4 +18,16 @@ defmodule Dwylbot.Rules.Helpers do
       :test -> test
     end
   end
+
+  @doc """
+  iex> Dwylbot.Rules.Helpers.label_member?([%{"id" => 1, "name"=> "lb"}], "lb")
+  true
+  iex> Dwylbot.Rules.Helpers.label_member?([%{"id" => 1, "name" => "lb"}], "er")
+  false
+  """
+  def label_member?(list_labels, label) do
+    list_labels
+    |> Enum.map(fn(l) -> l["name"] end)
```

Comment:
If your not going to be using the argument and want to write it shorthand, consider changing this:

```elixir
|> Enum.map(fn(l) -> l["name"] end)
```

to

```elixir
|> Enum.map(&(&1["name"]))
```

Initially it feels a bit messy, but after using it for a while, it's much nicer to work with